examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -106,28 +108,43 @@ async def raise_if_error_async(awaitable_call):
                         task=task, attribute=nidaqmx_types.TASK_ATTRIBUTE_NUM_CHANS)))
 
             number_of_channels = response.value
-
             async def read_data():
-                async for every_n_samples_response in every_n_samples_stream:
-                    await raise_if_error(every_n_samples_response)
-                    read_response: nidaqmx_types.ReadAnalogF64Response = await raise_if_error_async(
-                        client.ReadAnalogF64(
-                            nidaqmx_types.ReadAnalogF64Request(
-                                task=task,
-                                num_samps_per_chan=100,
-                                fill_mode=nidaqmx_types.GroupBy.GROUP_BY_GROUP_BY_CHANNEL,
-                                array_size_in_samps=number_of_channels * 100)))
-
-                    print(
-                        f"Acquired {len(read_response.read_array)} samples",
-                        f"({read_response.samps_per_chan_read} samples per channel)")
-                    print("Read Data:", read_response.read_array[:10])
+                samps_per_chan_read = 0
+
+                try:
```

Comment:
This doesn't diff very well unfortunately.

The changes are:
1. try/except for CancelledException. Otherwise the gather call will complete on the first cancel.
2. The 2 commented sections ensuring that the every-n-samples loop can cancel itself and that the done event will only cancel every-n-samples on error.